src/css/base/global.css

```diff
@@ -9,5 +9,7 @@ code {
 
 .component-container {
 	width: 50%;
-	margin: 0 auto;
+	margin: 0 auto 10px;
+	border-bottom: 1px solid var(--black);
```

Comment:
Maybe you can use 2px instead? Thus the container would be easily visible but not so large